packages/react-components/src/components/VideoGallery/HorizontalGallery.tsx

```diff
@@ -0,0 +1,208 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { DefaultButton, Icon, mergeStyles, Stack } from '@fluentui/react';
+import React, { useEffect, useMemo, useRef, useState } from 'react';
+import { OnRenderAvatarCallback, VideoGalleryRemoteParticipant, VideoStreamOptions } from '../../types';
+import { horizontalGalleryContainerStyle, leftRightButtonStyles } from '../styles/HorizontalGallery.styles';
+import { useContainerWidth, useIsSmallScreen } from '../utils/responsive';
+import { RemoteVideoTile } from './RemoteVideoTile';
+
+/**
+ * HorizontalGallery Component Props.
+ */
+export interface HorizontalGalleryProps {
+  participants: VideoGalleryRemoteParticipant[];
+  /** Remote videos view options */
+  remoteVideoViewOption?: VideoStreamOptions;
+  onCreateRemoteStreamView?: (userId: string, options?: VideoStreamOptions) => Promise<void>;
+  onDisposeRemoteStreamView?: (userId: string) => Promise<void>;
+  /** Callback to render a remote video tile */
+  onRenderRemoteVideoTile?: (remoteParticipant: VideoGalleryRemoteParticipant) => JSX.Element;
+  /** Callback to render a particpant avatar */
+  onRenderAvatar?: OnRenderAvatarCallback;
+  /**
+   * Whether to display a mute icon beside the user's display name.
+   * @defaultValue `true`
+   */
+  showMuteIndicator?: boolean;
+  /** Space to leave on the left of this gallery in pixels. */
+  leftGutter?: number;
+  /** Space to leave on the right of this gallery in pixels. */
+  rightGutter?: number;
+  /** Default `false`. If set to true, video tiles will not render remote video stream  */
+  hideRemoteVideoStream?: boolean;
+}
+
+/**
+ * Renders a horizontal gallery of video tiles.
+ * @param props - HorizontalGalleryProps {@link @azure/communication-react#HorizontalGalleryProps}
+ * @returns
+ */
+export const HorizontalGallery = (props: HorizontalGalleryProps): JSX.Element => {
+  const {
+    participants,
+    onCreateRemoteStreamView,
+    onDisposeRemoteStreamView,
+    onRenderRemoteVideoTile,
+    remoteVideoViewOption,
+    onRenderAvatar,
+    showMuteIndicator,
+    hideRemoteVideoStream,
+    leftGutter = 8,
+    rightGutter = 8
+  } = props;
+
+  const containerRef = useRef<HTMLDivElement>(null);
+  const isSmallScreen = useIsSmallScreen(containerRef);
+  const containerWidth = useContainerWidth(containerRef);
+  const [page, setPage] = useState(0);
+  const [maxTiles, setMaxTiles] = useState(0);
+  const [tileHeight, setTileHeight] = useState(120);
```

Comment:
There's a lot of values thrown around in this file: `120` `160` `88` - are these pixel values and do they represent small and large tiles?
Can we pull them out and rename accordingly, e.g
```
const TILE_SIZE_LARGE = { height: 120; width: 160 }; // Large tile size in rem
```
(convert px to rem^)